packages/dart/test/src/objects/parse_object_test.dart

```diff
@@ -420,5 +420,127 @@ void main() {
         verifyNoMoreInteractions(client);
       });
     });
+
+    group('update()', () {
```

Comment:
Why is it in the group in this section? I don't think it is necessary

Reply:
Every function in the ParseObject will have a group test. so that it will be easier to run tests for a specific function plus its more organamic I think.

Reply:
I got it right